doc/architecture/security.md

```diff
@@ -0,0 +1,64 @@
+Security
+========
+This document states which security issues that has been taken into consideration when developing this site.
+
+TLS
+---
+The connection from the client to the web server is made over TLS with a Amazon signed certificate.
+This is implemented on the Elastic Loadbalancer of Elastic Beanstalk.
+The request then propagates through our VPC (Virtual private cloud) in AWS as plain HTTP. 
+
+CSRF
+----
+The site has implemented protection against CSRF (Cross Site Request Forgery), 
+> this is a manin the middle attack that abuses the browser’s automatic cookie submission for cross-origin requests to issue state changing requests on the user’s behalf. 
```

Comment:
`manin` -> `man in`
Or maybe `man-in-the-middle`?

Reply:
Don't you like the sound of manin? It gives a certain feeling to it i would say. 😃
Will fix.